src/core_modules/capture-core/components/CardList/CardList.selectors.js

```diff
@@ -1,23 +1,23 @@
 // @flow
 import { createSelector } from 'reselect';
-import { chunk } from 'capture-core-utils';
 import { elementTypes } from '../../metaData/DataElement';
 
 
-const elementsSelector = props => props.dataElements;
+const elementsSelector = dataElements => dataElements;
 
 // $FlowFixMe[missing-annot] automated comment
 const makeElementsContainerSelector = () => createSelector(
     elementsSelector,
     (elements) => {
-        const newElements = [...elements];
         // $FlowFixMe[prop-missing] automated comment
         const imageDataElement = elements.find(a => a.type === elementTypes.IMAGE);
         if (imageDataElement) {
+            const newElements = [...elements];
             newElements.splice(newElements.indexOf(imageDataElement), 1);
         }
+
         return {
-            dataElementChunks: chunk(elements, 5),
+            dataElements: elements,
```

Comment:
Do we really need to have this here? I dont thing this PR is the right place to address this, but to my eyes this could be done a bit more gracefully. What you think?

Reply:
Yeah I didnt explain myself well @JoakimSM. I dont fully get if we need to return this `dataElements` from the selector. `CardList.component.js` is having as props the dataElements already.

Reply:
The imageDataElement that is potentially found here, should not be in the dataElements array that are passed on to the `CardListItem`, because we would like the image to display only once.